package.json

```diff
@@ -45,4 +45,4 @@
       "npx prettier --write"
     ]
   }
-}
+}
```

Comment:
```suggestion
}

```

Reply:
i wonder why the prettier autoformatter didn't fix hm